Lesson3.html

```diff
@@ -0,0 +1,33 @@
+<html>
+    <head>
+        <title>My Simple Html Page</title>
+        <link rel="stylesheet" type="text/css" href=/>
+    </head>
+    <body>
+        <p>How are You</p>
+        <button id="goodButton">Good😃</button>
+        <button id="badButton">Bad😞</button>
+        <button id ="awsomeButton">Awsome😍</button>
+        <p id ="answer"></p>
+        <script>
+            let goodButtonCmp = document.getElementById("goodButton");
+            let badButtonCmp =document.getElementById("badButton")
+            let awsomeButtonCmp = document.getElementById("awsomeButton")
+            let answerCmp = document.getElementById("answer");
+            function goodButtonClick(event){
+                answerCmp.textContent= "That's great 😃"
```

Comment:
do not omit `;` at the end of statement. Think about it as required to have.
Check your code for that missing semicolon.